src/App.jsx

```diff
@@ -1,17 +1,26 @@
 import './App.css';
+import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculatorb from './components/Calculator';
 import RandomQuote from './components/quote';
+import Home from './components/Home';
 
 function App() {
   return (
     <div className="App">
-      <div className="Title-container">
-        <h1>Math-Magicians</h1>
-      </div>
-      <div className="appContainer">
-        <Calculatorb />
-        <RandomQuote />
-      </div>
+      <header className="Header-Container">
+        <h1 className="Tittle"> Math Magicians</h1>
+        <ul>
+          <Link to="/"><li className="menu">Home</li></Link>
+          <Link to="/calculator"><li className="menu">Calculator</li></Link>
+          <Link to="/quote"><li className="menu">Quote</li></Link>
+        </ul>
+      </header>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculatorb />} />
+        <Route path="/quote" element={<RandomQuote />} />
+      </Routes>
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- [ ] Looking at the video below, you can see that when I tried starting your app, I got an error in the browser related to `react-router-dom` not found. You are using the react router dom but it's not in your dependency array, I'll suggest you add it to your project by running this cmd 
```sh
npm i react-router-dom
```
:smile: :wave: 

- - Video:

https://github.com/CarlosZ96/mathmagicians/assets/107632780/1afbf1ba-4943-485a-967a-2dc92754da19